scripts/scripts.js

```diff
@@ -990,6 +990,30 @@ async function loadEager(doc) {
   }
 }
 
+export function getLanguage(locale) {
+  const langs = {
+    us: 'en-US',
+    fr: 'fr-FR',
+  };
+
+  let language = langs[locale];
+  if (!language) language = 'en-US';
+
+  return language;
+}
```

Comment:
I think having a `getLanguage`, `getLocale`, _and_ a [`isFr`](https://github.com/hlxsites/pga-presidents-cup/blob/8f0c08ab10adf4c871224fea5f1168ca1abfb746/scripts/scripts.js#L829) function could be a little confusing. Is there a way to extend `getLanguage` or `getLocale` to absorb the limited functionality of `isFr` to replace it?

Reply:
So you confirm for now there are only EN/FR? And hence, following Franklin principles, we should only implement for the actual use case?

Reply:
For now I simplified the implementation to only support en/fr languages, in [4e5f0d5](https://github.com/hlxsites/pga-presidents-cup/pull/3/commits/4e5f0d560ff5bc156d2cc038380b38c9b052aa9c)